### ISSUE TITLE
is this maintained?

### ISSUE THREAD

**Issue by abubelinha** [lnielsen/zenodo-uploader#1]
Hello
I read about this package in Agosti & al. 2019 (https://doi.org/10.5281/ZENODO.3346782)

As there were some [Zenodo changes in late 2023](https://help.zenodo.org/docs/about/whats-changed/) I would like to ask if this software is still being usable or there are other Python alternatives for the same purpose.

Thanks a lot
@abubelinha

**Comment by matdillen** [comment]
No, I don't believe this repo will ever be updated. You might find a more recent alternative in https://github.com/plazi/lycophron
